File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Invoices/IInvoiceData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Invoices
{
    public interface IInvoiceData :  CoreObject.IObjectID, CoreObject.IObjectTitle, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IInvoiceTotalPrice, IInvoiceDate
    {

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Invoices/IInvoiceData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Invoices;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IInvoiceData extends IInvoiceDate,IInvoiceTotalPrice,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
